qa/workunits/mon/crush_ops.sh

```diff
@@ -14,6 +14,7 @@ ceph osd crush dump
 ceph osd crush rule dump
 ceph osd crush rule ls
 ceph osd crush rule list
+ceph osd crush rule ls
```

Comment:
please drop this duplicated test, see https://github.com/ceph/ceph/pull/16423/files#diff-bfe2ac67a1e18313cc7532a6a24dbad6R15